src/target/ext/host_io.rs

```diff
@@ -269,9 +285,9 @@ define_ext!(HostIoOpenOps, HostIoOpen);
 
 /// Nested Target Extension - Host I/O close operation.
 pub trait HostIoClose: HostIo {
-    /// Close the open file corresponding to fd and return 0, or
+    /// Close the open file corresponding to fd and return Ok(()), or
     /// [`HostIoError::Errno`] if an error occurs.
```

Comment:
I know I'm being a real stickler for docs and wording, but it's important to get right - sorry 😅

`HostIoResult<(), Self>` is actually self-describing. The only option for success is returning `Ok(())`, and the only option for failure is returning `HostIoError::Errno`.

Then the return type is _not_ `()`, that's when you'd want to include some extra docs explaining what the value represents. i.e: if you're returning an `fd` as a `u32`, that's when you'd include that comment.

In general though, you can go ahead and remove the lines that mention "return [`HostIoError::Errno`] if an error occurs."